### PR TITLE
Upgrade to >v5 xml at parser handling level

### PR DESCRIPTION
**Pull request by bethac07**
Thanks to @sugan89 for creating the files for testing and working a lot on the logic of this!

### CODE REVIEW

**Comment by ErinWeisbart** [comment]
I tested data read off S3 (cellpainting-gallery)
<details><summary>with command</summary>

``` 
pe2loaddata ../../config.yaml /Users/eweisbar/Desktop/2024_11_15_OASIS_HepaRG_AD/BR00145690/load_data_with.csv \
--index-directory "s3://cellpainting-gallery/cpg0033-oasis-pilot/broad/images/2024_11_15_OASIS_HepaRG_AD/images/BR00145690" \
--index-file "s3://cellpainting-gallery/cpg0033-oasis-pilot/broad/images/2024_11_15_OASIS_HepaRG_AD/images/BR00145690__2024-11-08T21_30_25-Measurement1/Images/Index.xml" \
--illum --illum-directory cpg0033-oasis-pilot/broad/images/2024_11_15_OASIS_HepaRG_AD/illum/BR00145690 \
--plate-id BR00145690 \
--illum-output /Users/eweisbar/Desktop/2024_11_15_OASIS_HepaRG_AD/BR00145690/load_data_with_illum.csv \
--sub-string-out cpg0033-oasis-pilot --sub-string-in /home/ubuntu/bucket/cpg0033-oasis-pilot \
--search-subdirectories
```

</details>

<details><summary>with config</summary>

```
channels:
    HOECHST 33342: OrigDNA
    Alexa 568: OrigAGP
    Alexa 647: OrigMito
    Alexa 488: OrigER
    Alexa 488 Long (CP): OrigRNA
    Brightfield high: OrigBrightfield
metadata:
    Row: Row
    Col: Col
    FieldID: FieldID
    PlaneID: PlaneID
    ChannelID: ChannelID
    ChannelName: ChannelName
    ImageResolutionX: ImageResolutionX
    ImageResolutionY: ImageResolutionY
    ImageSizeX: ImageSizeX
    ImageSizeY: ImageSizeY
    BinningX: BinningX
    BinningY: BinningY
    MaxIntensity: MaxIntensity
    PositionX: PositionX
    PositionY: PositionY
    PositionZ: PositionZ
    AbsPositionZ: AbsPositionZ
    AbsTime: AbsTime
    MainExcitationWavelength: MainExcitationWavelength
    MainEmissionWavelength: MainEmissionWavelength
    ObjectiveMagnification: ObjectiveMagnification
    ObjectiveNA: ObjectiveNA
    ExposureTime: ExposureTime
```

</details>

and it appears to have worked!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # pe2loaddata
-Script to parse a Phenix metadata XML file and generate a .CSV for CellProfiler's loaddata module
+Script to parse a Phenix metadata XML file and generate a .CSV for CellProfiler's loaddata module.
+Supports XML files made by Harmony versions V5 and V7.
```
Comment (sugan89) [inline README.md]: Should we mention it supports V6 as well?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # pe2loaddata
 Script to parse a Phenix metadata XML file and generate a .CSV for CellProfiler's loaddata module.
-Supports XML files made by Harmony versions V5 and V7.
+Tested for XML files made by Harmony versions V1, V5, and V7, but as far as we know supports all.
```
Comment (sugan89) [inline README.md:3]: Sorry! Just noticed this change.

**Review by ErinWeisbart — APPROVED** [review]
LGTM!

File: README.md (hunk 2 above)
Comment (bethac07) [reply README.md:3]: All good :)

**Pull request merged**